float64[] x
float64[] y
float64[] z
int32[] indices
int32 frame_number
